pages/home/EmailSignupForm.jsx

```diff
@@ -20,10 +20,10 @@ var PledgeSignupForm = React.createClass({
       return;
     }
 
-    if (process.env.NODE_ENV !== 'production' && !process.env.PLEDGE_MAILINGLIST_URL) {
+    if (process.env.NODE_ENV !== 'production' && !process.env.NEWSLETTER_MAILINGLIST_URL) {
       e.preventDefault();
-      alert("PLEDGE_MAILINGLIST_URL is not defined. Simulating a successful pledge signup now.");
-      window.location = "?pledge=thanks";
+      alert("NEWSLETTER_MAILINGLIST_URL is not defined. Simulating a successful newsletter signup now.");
+      window.location = "?signup=thanks";
```

Comment:
Hey @alicoding , can we chat over this tomorrow (Tue) morning? I realize I can't just mimic what you did for Advocacy site since I don't know 
1) which ET list I'm asking users to sign up (is it the `mozilla-learning-network` list? Should we double check with people on the MLN program know before we do the switching? )
2) what fields/params to send 
3) how the Thank You redirect work :confused: